src/app/[locale]/projects/detail/[id]/components/ProjectDetailTab.tsx

```diff
@@ -0,0 +1,269 @@
+// Copyright (C) Siemens AG, 2023. Part of the SW360 Frontend Project.
+
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+
+// SPDX-License-Identifier: EPL-2.0
+// License-Filename: LICENSE
+
+"use client"
+
+import { Col, Row, ListGroup, Tab, Button, Dropdown } from 'react-bootstrap'
+import Summary from './Summary'
+import React, { useState, useEffect } from 'react'
+import PageSpinner from '@/components/Spinner/Spinner'
+
+import { useTranslations } from 'next-intl'
+import { COMMON_NAMESPACE } from '@/object-types/Constants'
+
+import ApiUtils from "@/utils/api/api.util"
+import CommonUtils from '@/utils/common.utils'
+import { Session } from '@/object-types/Session'
+import HttpStatus from '@/object-types/enums/HttpStatus'
+import { signOut } from 'next-auth/react'
+
+import { SummaryDataType } from '@/object-types/SummaryDataType'
+import { notFound } from 'next/navigation'
+
+export default function ViewProjects({ session, projectId }: { session: Session, projectId: string }) {
+
+    const t = useTranslations(COMMON_NAMESPACE)
+    const [loading, setLoading] = useState(true)
+    const [summaryData, setSummaryData] = useState<SummaryDataType>(
+        {
```

Comment:
You can set init value for summaryData is **undefined** then detail component is only shown when the summary data is not **undefined**

Reply:
Yep! Sounds good!